examples/qfiae/utils_qfiae.py

```diff
@@ -0,0 +1,108 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from qibo import gates
+from qibo.models import Circuit
+from qibo.models.iqae import IQAE
+
+
+def fourier_series(coeffs, x_val, period=2 * np.pi):
+    """Computes the Fourier series for a given set of coefficients in exponencial form."""
+
+    series = [complex(0, 0) for _ in range(len(x_val))]

```

Comment:
I think here you can use something more clear, like:

```python
xval_len = len(x_val)
series = np.zeros(xval_len, dtype=np.complex128)
```

and, moreover, it seems to me you can compress the two `for` cycles into one by summing in one shot the exponential with `+n` and `-n`.